packages/@react-aria/checkbox/src/useCheckboxGroup.ts

```diff
@@ -33,12 +37,14 @@ export interface CheckboxGroupAria {
 export function useCheckboxGroup(props: AriaCheckboxGroupProps, state: CheckboxGroupState): CheckboxGroupAria {
   let {isDisabled, name} = props;
 
-  let {labelProps, fieldProps} = useLabel({
+  let {labelProps, fieldProps, descriptionProps, errorMessageProps} = useField({
     ...props,
     // Checkbox group is not an HTML input element so it
     // shouldn't be labeled by a <label> element.
     labelElementType: 'span'
   });
+  checkboxGroupDescriptionIds.set(state, descriptionProps.id);
```

Comment:
if anyone alters the state after it is returned by its hook, say maybe to `{...state, foo: true}` spread it into a bigger state object, this will break
I know we've done this pattern in the past, but maybe it'd make more sense to set a field on state that people are unlikely to override?

happy for this to be a discussion and followup

Reply:
let's not block this pr. we use this pattern in many, many places, and this has never come up.